nautobot/docs/user-guide/administration/installation/index.md

```diff
@@ -1,24 +1,98 @@
+
+=== "Docker Compose"
+
+    Nautobot can be deployed via Docker using [Nautobot Docker images](../guides/docker.md) that are available either from:
+
+    * [Docker Hub](https://hub.docker.com/r/networktocode/nautobot)
+    * [GitHub Container Registry](https://github.com/nautobot/nautobot/pkgs/container/nautobot) 
+
+    For local testing and lab work you can use the Nautobot [Docker Compose](https://github.com/nautobot/nautobot-docker-compose) project which leverages the container images above. 
+
+    For more information about the Docker tags, Docker configurations, or using container images for your own development environment, see [Nautobot in Docker](../guides/docker.md).
+
+    <h3>Docker Compose Pros</h3>
+
+    * Quick to deploy for single node lab work and testing
+    * For those looking for a multi-container single-node install for Nautobot often coupled with backup & HA capabilities from their hypervisor manager
+
+    <h3>Docker Compose Cons</h3>
+
+    * Docker compose on its own is missing advanced features like automated rollbacks, self-healing applications, and load balancing 
+    * Limited scaling if on single host 
+
+=== "Kubernetes Helm Charts"
+
+    You can [install Nautobot via Helm Charts](https://docs.nautobot.com/projects/helm-charts/en/stable/) for Kubernetes documentation. 
+
+    If you are looking for an example of what that process looks like in a development environment using `minikube`, you can check out the three part blog series covering deploying Nautobot to Kubernetes:
+        
+    * [Part 1](https://networktocode.com/blog/deploying-nautobot-to-kubernetes-01/)
+    * [Part 2](https://networktocode.com/blog/deploying-nautobot-to-kubernetes-02/)
+    * [Part 3](https://networktocode.com/blog/deploying-nautobot-to-kubernetes-03/)
+
+    <h3>Helm Charts Pros</h3>
+
+    * Helm charts serves as a package manager for Kubernetes, streamlining the deployment of complex applications within Kubernetes clusters
+    * Using Kubernetes features, you can manage large-scale deployments efficiently and ensure high-availability for production grade systems
+
+    <h3>Helm Charts Cons</h3>
+
+    * Requires knowledge of Kubernetes and has a steeper learning curve for setting up
+    * Kubernetes clusters can use a lot of resources and monitoring effort
+
+=== "Nautobot Cloud"
+
+    [Nautobot Cloud](https://networktocode.com/nautobot/nautobot-cloud/) is a commercial Software as a Service (SaaS) offering from Network to Code. It offers self-service and turnkey deployment for Nautobot removing the need for infrastructure setup and maintenance for the application. 
+
+    Nautobot Cloud also streamlines the deployment and management of several other commonly used automation tools:
+    * [Ansible AWX](https://github.com/ansible/awx)
+    * [Batfish](https://github.com/batfish/batfish)
+    * Commonly used network telemetry stack (Grafana / Prometheus / Telegraf)
+    * Linux automation utilities
```

Comment:
```suggestion
    * [Ansible AWX](https://github.com/ansible/awx)  
    * [Batfish](https://github.com/batfish/batfish)  
    * Commonly used network telemetry stack (Grafana / Prometheus / Telegraf)  
    * Linux automation utilities  
```

Reply:
@jvanderaa I think the suggestion should include a new line before.

Reply:
Agreed, just added a newline, just like the list for Heml